src/Analyzer.JsonRuleEngine.UnitTests/InOperatorTests.cs

```diff
@@ -36,6 +36,32 @@ public void EvaluateExpression_ValidDataType_ReturnsExpectedEvaluationResult(boo
             Assert.AreEqual(evaluationResult, inOperator.EvaluateExpression(desiredValueJToken));
         }
 
+        [DataTestMethod]
```

Comment:
Instead of duplicating the whole test, can't we also create an inOperator with `isNegative: true` in the original test function and do both validations there?

Reply:
I think the same comment applies for other operators tests